client/lib/with-dimensions/index.jsx

```diff
@@ -48,7 +50,11 @@ export default ( EnhancedComponent, { domTarget } = {} ) => class WithWidth exte
 	render() {
 		return (
 			<div ref={ this.handleMount }>
-				<EnhancedComponent { ...this.props } width={ this.state.width } />
+				<EnhancedComponent
+					{ ...this.props }
+					width={ this.state.width }
```

Comment:
you mean add them to the `SpacerDiv`?  They don't belong inside `withDimensions`